icubam/predicu/plot/__init__.py

```diff
@@ -103,74 +113,60 @@ def plot_int(
     PLOTS.append(plot_name)
 
 
-def plot(plot_name, data, **plot_args):
+def plot(
+  plot_name: str,
+  cached_data: Dict[str, pd.DataFrame],
+  output_dir: str = DEFAULTS["output_dir"],
+  output_type: str = DEFAULTS["output_type"],
+  api_key: str = DEFAULTS["api_key"],
+  matplotlib_style: str = DEFAULTS["matplotlib_style"],
+):
   plot_module = __import__(f"{plot_name}", globals(), locals(), ["plot"], 1)
```

Comment:
In the long term, I think we should rather manually import all plot modules at the top of `__init__`, e.g.
```oy
from . import barplot_deaths_per_day
```
and then use something like,
```py
plot_module = globals()[plot_name]
```
inside this function. Import of a user define path is bad, as it's not far from arbitrary code execution as long as one is able to put some file in that folder.

Reply:
agreed. let's fix that in another PR?